requirements-dev.txt

```diff
@@ -8,7 +8,7 @@ pytest-cov==2.5.1
 flaky==3.4.0
 hypothesis==3.44.14
 grequests==0.3.0
-detox==0.11
+tox==2.9.1
```

Comment:
we could also remove this dependency, since it's not really used by any of our scripts

Reply:
Ok